### PR TITLE
Install fonts, locales and use gosu to allow root access

### PR DESCRIPTION
**Pull request by colinnewell**
In our use of Odoo in a docker container we found these features were generally necessary.

Locale is set to en_GB currently.

### CODE REVIEW

**Comment by mlaitinen** [comment]
Adding locales and fonts is an excellent idea.

Did you notice that gosu was used before but was later removed? Can't really say why. https://github.com/odoo/docker/commit/2b5021a8becc248f38ba1cde79563022dbff0b83

**Comment by colinnewell** [comment]
@mlaitinen I hadn't noticed that.  I did wonder if it might be controversial but it is something we've found to be useful so I figured I'd make sure it was in a separate commit and see what people said.

**Comment by CraigBarnesOpusVL** [comment]
looking at the commit prior to gosu removal it was not possible to run anything but odoo at run time, now it is.  

Also the  docker exec command did not exist then.

**Comment by aleks-v-k** [comment]
**+1** for using gosu instead of USER directive. We (kuberdock team http://kuberdock.com/) need root to chown /var/lib/odoo which we mount as Persistent Drive in kubernetes (https://github.com/kubernetes/kubernetes/blob/master/docs/design/persistent-storage.md)
Also it is needed in 9.0/*.

**Comment by rimusz** [comment]
+1

File: 8.0/Dockerfile [hunk 1]
```diff
@@ -30,11 +45,21 @@ RUN set -x; \
...
+
 # Copy entrypoint script and Odoo configuration file
 COPY ./entrypoint.sh /
 COPY ./openerp-server.conf /etc/odoo/
 RUN chown odoo /etc/odoo/openerp-server.conf
 
+RUN locale-gen en_GB.UTF-8
+ENV LANG en_GB.UTF-8
+ENV LANGUAGE en_GB:en
+ENV LC_ALL en_GB.UTF-8
```
Comment (Yajo) [inline 8.0/Dockerfile]: Odoo's core language is en_US AFAIK, so that should be the default IMHO.

**Comment by sle-odoo** [comment]
Hello,
Sorry to comment this late on this pull request. These images have diverged so much, i don't know what we can still merge of it. Some of your changes made sense, like installing the fonts for reportlab, i'll definitely to that soon. We do not want to use gosu at this point, we don't really need it. You can now exec to the container  as any user you want or make the needed changes in the dockerfile as the user you want.

I'm closing this one, but if you're still interested in contributing i'll definitely check your changes in a timely manner.

Regards

**Pull request closed without merging**